CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Update the `dump` function to handle addresses: PR [#175](https://github.com/tact-lang/tact/pull/175)
 - The implicit empty `init` function is now present by default in the contract if not declared: PR [#167](https://github.com/tact-lang/tact/pull/167)
+- Add JSON Schema for `tact.config.json`: PR [#194](https://github.com/tact-lang/tact/pull/194)
```

Comment:
Should we move it to the `### Added subsection`? And then the word "Add" at the beginning is redundant

Reply:
You are correct. Fixed this